<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>551</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDJadwalLabel">
      <property name="text">
       <string>ID Jadwal</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="EditJadwal"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="iDPoliklinikLabel">
      <property name="text">
       <string>ID Poliklinik</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EditPoli"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="iDPemeriksaLabel">
      <property name="text">
       <string>ID Pemeriksa</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EditPeriksa"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="jamMulaiLabel">
      <property name="text">
       <string>Jam Mulai</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTimeEdit" name="TimeMulai"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="jamSelesaiLabel">
      <property name="text">
       <string>Jam Selesai</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTimeEdit" name="EditSelesai"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="hariLabel">
      <property name="text">
       <string>Hari</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cmbHari">
      <item>
       <property name="text">
        <string>Senin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Selasa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rabu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kamis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jum'at</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sabtu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minggu</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCari">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>551</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>551</width>
     <height>191</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>93</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID Jadwal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID Poliklinik</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID Pemeriksa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jam Mulai</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jam Selesai</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hari</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
